_episodes/13-looking-up-data.md

```diff
@@ -163,3 +158,9 @@ As it returns the whole row for each match, you can use a transformation to extr
 You can use this function to compare the contents of two OpenRefine projects, or to use data between the two projects.
 
 The VIB-Bits extension adds a number of very useful functions to OpenRefine. One of them is "Add column(s) from other projects...", which provides a dialog window to help you work with the `cross` function with less typing.
+
+## Extensions
```

Comment:
This line needs to start with a > character so that it is part of the callout (otherwise the heading ends up separate to the text in the callout)

Reply:
I'm going to try to guess which line you mean.  This comment is appearing between 162 and 163, so I'll add the angle bracket before the hashtags before the word Extensions.